pkg/motor/resource.go

```diff
@@ -104,45 +95,3 @@ func (r *motorResources) GetSchema(did string) (*st.WhatIs, *st.SchemaDefinition
 
 	return nil, nil, false
 }
-
-func (r *motorResources) GetWhereIs(ctx context.Context, did string, address string) (*bt.WhereIs, error) {
```

Comment:
Why are these being removed?

Reply:
these are 1:1 with the handleBucketQuery and handleBucketGroupQuery operations, they were not checking from the underlying maps

Reply:
the reason for that was that they where the source of truth for the cache. so querying would always override the cache as to keep things up to date